runtime/compute/ARMComputeEx/src/core/CL/CLKernelLibrary.cpp

```diff
@@ -258,7 +258,8 @@ Kernel CLKernelLibraryEx::create_kernel(const std::string &kernel_name,
   return Kernel(kernel_name, cl_program);
 }
 
-void CLKernelLibraryEx::add_built_program(std::string_view built_program_name, cl::Program program)
+void CLKernelLibraryEx::add_built_program(const std::string &built_program_name,
+                                          cl::Program program)
```

Comment:
Since `_built_programs_map` is returned via a member function as `const std::map<std::string, cl::Program>&`, I couldn't use a transparent comparator. Therefore, I reverted the use of `std::string_view` here.